activesupport/test/core_ext/duration_test.rb

```diff
@@ -66,6 +66,12 @@ def test_inspect
     assert_equal "1 month and 1 day",               (1.month + 1.day).inspect
     assert_equal "6 months and -2 days",            (6.months - 2.days).inspect
     assert_equal "10 seconds",                      10.seconds.inspect
+    assert_equal "6 minutes and 40 seconds",        (5.minutes + 100.seconds).inspect
+    assert_equal "6 hours and 40 minutes",          (5.hours + 100.minutes).inspect
+    assert_equal "1 day and 1 hour",                25.hours.inspect
```

Comment:
Actually `1.day` and `24.hours` behaves differently under DST affected, but both inspect says `"1.day"` makes hard to debug what is happened, so I don't want this change. Closing this for now.